packages/@ember/-internals/deprecations/index.ts

```diff
@@ -111,7 +111,7 @@ export const DEPRECATIONS = {
   }),
   DEPRECATE_TEMPLATE_ACTION: deprecation({
     id: 'template-action',
-    url: 'https://deprecations.emberjs.com/id/template-action',
```

Comment:
These id urls were supposed to work. Hmm

Ah, the url should be: https://deprecations.emberjs.com/id/deprecate-template-action/

Reply:
ah cool... havn't known that... now updated